src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -148,10 +149,9 @@
                         "jruby-puppet.max-active-instances.")}))
      (when (jruby-schemas/shutdown-poison-pill? pool-instance#)
        (jruby/return-instance ~jruby-service pool-instance# ~reason)
-       (sling/throw+
-        {:type    ::service-unavailable
-         :message (str "Attempted to borrow a JRuby instance from the pool "
-                       "during a shutdown. Please try again.")}))
+       (mw/throw-service-unavailable!
```

Comment:
Given that this project is still pre-1.0 (and likely only used by other puppetlabs projects) (and these functions are brand-spankin'-new) I would like to avoid deprecation and just move them.

The ceremony of a release is pretty much the same regardless of X, Y, or Z:
* update version number (if necessary)
* update changelog
* run jenkins job
* send announcement